test/parallel_api/numeric/numeric.ops/exclusive_scan_by_segment.pass.cpp

```diff
@@ -29,10 +29,6 @@ using namespace oneapi::dpl::execution;
 #endif // TEST_DPCPP_BACKEND_PRESENT
 using namespace TestUtils;
 
-// This macro may be used to analyze source data and test results in test_exclusive_scan_by_segment
-// WARNING: in the case of using this macro debug output is very large.
-// #define DUMP_CHECK_RESULTS
-
```

Comment:
(optional)
I have the same comment for all these "dump" blocks. 
My preference would be to either leave them alone or improve them in some unified way.